docs/source/usage/reference.rst

```diff
@@ -50,3 +50,240 @@ Thank you and enjoy PIConGPU and our community!
 .. raw:: html
 
    <iframe src="https://computationalradiationphysics.github.io/picongpu-communitymap/" style="width:100%; height: 650px;" frameborder="0"></iframe>
+
+
+Publications
+------------
+
+The following publications are sorted by topics.
+Papers covering multiple topics will be listed multiple times.
+In the end, a list of all publications in chronological order is given with more details.
+If you want to add your publication to the list as well please feel free to contact us or open a pull request directly. 
+
+Application of PIConGPU in various physics scenarios
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In the following, a list of publications describing using PIConGPU is given in various cases.
+
+Laser plasma acceleration
+"""""""""""""""""""""""""
+
+ - Laser wakefield acceleration (LWFA) with self-truncated ionization-injection (STII)
+ - Ion acceleration from droplet iradiation
+ - Laser-driven proton beam structering
+ - Hybrid laser-driven/beam-driven plasms acceleration 
+ - Radiation from LWFA (concept paper)
+   
```

Comment:
Just dumping this quickly:
- Debus PRX 2019
- Huebl PPCF 2020
- Hilz Nat. Comm. 2018 (below: `Hilz2018`)
- Obst-Huebl Nat. Comm 2018
- Couperus Nat. Comm. 2017
- Obst Sci. Rep. 2017
- Schramm J. Phy.s Conf.Ser. 2017
- Fortmann-Grtoe: NOBUGS Conf. 2016, SPIE 2017

Reply:
@ax3l I assume with *Schramm J. Phy.s Conf.Ser. 2017* you mean https://doi.org/10.1088/1742-6596/874/1/012028, is this correct? There are no PIConGPU results presented/discussed in there. Why did you want to reference this paper?

Reply:
@ax3l reagrding the 
> Simulations of single-particle imaging of hydrated proteins with x-ray free-electron lasers
> Fortmann-Grote, C ; Bielecki, J ; Jurek, Z ; Santra, R ; Ziaja-Motyka, B ; Mancuso, A. P ; Sawhney, Kawal ; Chubar, Oleg 
> Proceedings of SPIE - The International Society for Optical Engineering, 2017-08-23, Vol.10388, p.103880M-103880M-22

I do not have access to this SPIE journal. Could you please describe how PIConGPU was used for this paper?

Reply:
Oh, must have listed by accident. I wasn't sure anymore what we added in the J. Phy.s Conf.Ser., thanks for checking.

Proceedings of SPIE: also listed by accident, that's a simex_platform follow-up paper unrelated to PIConGPU.